<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="nuglaf" failures="4" tests="25" name="Sprint2" time="33.518" errors="0" timestamp="06 Jan 2021 05:48:17 GMT">
  <testcase classname="TestNG.TestNG.ExceptionTimeoutTest" name="testCase1" time="0.013"/>
  <testcase classname="TestNG.TestNG.ExceptionTimeoutTest" name="testCase2" time="0.003">
    <failure type="org.testng.TestException" message="
Method ExceptionTimeoutTest.testCase2()[pri:0, instance:TestNG.TestNG.ExceptionTimeoutTest@4b8729ff] should have thrown an exception of type class java.lang.NumberFormatException">
      <![CDATA[org.testng.TestException: 
Method ExceptionTimeoutTest.testCase2()[pri:0, instance:TestNG.TestNG.ExceptionTimeoutTest@4b8729ff] should have thrown an exception of type class java.lang.NumberFormatException
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- testCase2 -->
  <testcase classname="TestNG.TestNG.GoogleTitleTest" name="login" time="0.007">
    <failure type="junit.framework.ComparisonFailure" message="Google expected:&amp;lt;[Google]&amp;gt; but was:&amp;lt;[Title is not matched]&amp;gt;">
      <![CDATA[junit.framework.ComparisonFailure: Google expected:<[Google]> but was:<[Title is not matched]>
at junit.framework.Assert.assertEquals(Assert.java:100)
at TestNG.TestNG.GoogleTitleTest.login(GoogleTitleTest.java:35)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase classname="TestNG.TestNG.TestNGFeatures" name="loginTest" time="0.002">
    <failure type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
at TestNG.TestNG.TestNGFeatures.loginTest(TestNGFeatures.java:13)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase classname="TestNG.TestNG.testCase2" name="Checkout" time="0.002"/>
  <testcase classname="TestNG.TestNG.testCase2" name="SelectItem" time="0.001"/>
  <testcase classname="TestNG.TestNG.testCase2" name="login" time="0.001"/>
  <testcase classname="TestNG.TestNG.TestNGFeatures" name="HomePage" time="0.0">
    <skipped/>
  </testcase> <!-- HomePage -->
  <testcase classname="TestNG.TestNG.TestNGFeatures" name="RealPage" time="0.0">
    <skipped/>
  </testcase> <!-- RealPage -->
  <testcase classname="TestNG.TestNG.TestNGFeatures" name="SearchPage" time="0.0">
    <skipped/>
  </testcase> <!-- SearchPage -->
  <testcase classname="TestNG.TestNG.InvocationCountTest" name="sum" time="0.002"/>
  <testcase classname="TestNG.TestNG.InvocationCountTest" name="sum" time="0.002"/>
  <testcase classname="TestNG.TestNG.InvocationCountTest" name="sum" time="0.0"/>
  <testcase classname="TestNG.TestNG.InvocationCountTest" name="sum" time="0.001"/>
  <testcase classname="TestNG.TestNG.InvocationCountTest" name="sum" time="0.001"/>
  <testcase classname="TestNG.TestNG.InvocationCountTest" name="sum" time="0.001"/>
  <testcase classname="TestNG.TestNG.InvocationCountTest" name="sum" time="0.002"/>
  <testcase classname="TestNG.TestNG.InvocationCountTest" name="sum" time="0.003"/>
  <testcase classname="TestNG.TestNG.InvocationCountTest" name="sum" time="0.001"/>
  <testcase classname="TestNG.TestNG.InvocationCountTest" name="sum" time="0.002"/>
  <testcase classname="TestNG.TestNG.dependency" name="method1" time="0.002">
    <failure type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
at TestNG.TestNG.dependency.method1(dependency.java:12)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- method1 -->
  <testcase classname="TestNG.TestNG.dependency" name="method2" time="0.0">
    <skipped/>
  </testcase> <!-- method2 -->
  <testcase classname="TestNG.TestNG.testCase1" name="login" time="0.006"/>
  <testcase classname="TestNG.TestNG.testCase1" name="linkText" time="0.088"/>
  <testcase classname="TestNG.TestNG.testCase1" name="Alert" time="0.051"/>
</testsuite> <!-- Sprint2 -->
